test_runner/regress/test_subscriber_branching.py

```diff
@@ -0,0 +1,216 @@
+
+from __future__ import annotations
+
+import threading
+import time
+
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import NeonEnv, logical_replication_sync
+from fixtures.utils import query_scalar, wait_until
+
+
+# This test checks that branching of timeline with logical subscriptions does not affect logical replication.
+# Endpoint on a new branch will start with `disable_logical_replication_subscribers` set to True
+# this will prevent it from receiving any changes until it is set to False
+# Users can later set it to False explicitly, if they want to receive changes on a new branch.
```

Comment:
Re-read this today after a break and now I think, what will be the final UX?

1. Control plane will always set `[neon.]disable_logical_replication_subscribers` for branches?
2. But this means that inbound LR could be only set up in the root branch, so if you haven't had LR set up in the root, you won't be able to easily set it up in the branch

How we will allow LR in the branch? Will it be some cplane API? It can be probably some branch flag

What was your thinking?